### PR TITLE
[MGTL] Allow multiple bulk element ids in identify

### PR DESCRIPTION
**Pull request by endJunction**
Upon request for Dirichlet BCs inside of the domain defined along lines.

@Thomas-TK Could you provide an example?

### CODE REVIEW

**Comment by Thomas-TK** [comment]
Of course, it's my pleasure !

**Comment by Thomas-TK** [comment]
Here we go: an unstructured triangle mesh and a Y-line going through the domain (typical for river BCs).  Additionally, an according prism mesh. 
Thank you 
Thomas

[SimpleRiverExample.zip](https://github.com/ufz/ogs/files/2538198/SimpleRiverExample.zip)


PS: that's how it looks like
riverpointallocation

File: MeshGeoToolsLib/IdentifySubdomainMesh.cpp [hunk 1]
```diff
@@ -133,21 +133,8 @@ std::vector<std::size_t> identifySubdomainMeshElements(
             OGS_FATAL(
                 "Expect exactly one element to be found in the bulk mesh.");
```
Comment (TomFischer) [inline MeshGeoToolsLib/IdentifySubdomainMesh.cpp]: Is the error message still correct?

**Comment by Thomas-TK** [comment]
Looks good with triangle mesh. . .will test it for prism meshes as well 
riverpointallocationresult

File: MeshGeoToolsLib/IdentifySubdomainMesh.cpp (hunk 1 above)
Comment (endJunction) [reply MeshGeoToolsLib/IdentifySubdomainMesh.cpp]: Fixed: "exactly one" to at "least one".

**Comment by Thomas-TK** [comment]
I works for the 3D prism model as well. !!!!

**Comment by Thomas-TK** [comment]
I tested triangles, line elements and individual points as  elements on a triangle and prism mesh. Works very well as BCs inside the domain. 
riverpointallocationresultprism

**Comment by endJunction** [comment]
Put it in a ctest and add some docu on the benchmark page. Then, and only then, can you rely on these features being correct in the feature.

**Pull request merged**